tests/configs/config_ds000246.py

```diff
@@ -1,12 +1,16 @@
 """
-Auditory MEG
+Brainstorm - Auditory Dataset.
+
+See https://openneuro.org/datasets/ds000246/versions/1.0.0 for more
+information.
 """
 
 study_name = 'ds000246'
 bids_root = '~/mne_data/ds000246'
 deriv_root = '~/mne_data/derivatives/mne-bids-pipeline/ds000246'
 
 runs = ['01']
+crop_runs = (0, 180)  # Reduce memory usage on CI system
```

Comment:
We usually have to crop CTF data in our MNE examples. I'm hoping that cropping it a bit here will similarly help